rotkehlchen/api/v1/schemas.py

```diff
@@ -3027,3 +3027,136 @@ def make_history_event_filter(
 
 class EthStakingHistoryStatsDetails(EthStakingHistoryStats, AsyncIgnoreCacheQueryArgumentSchema):
     """Schema for querying ethereum staking history details"""
+
+
+class ExportHistoryEventSchema(TimestampRangeSchema, DBOrderBySchema):
+    """Schema for quering history events"""
+    exclude_ignored_assets = fields.Boolean(load_default=True)
+    event_identifiers = DelimitedOrNormalList(fields.String(), load_default=None)
```

Comment:
Don't duplicate code. This filters already exists in other schema so just inherit ir or use it completly. Not sure if there is need to create a new one

Reply:
The schema that has all these is the `HistoryEventSchema`, but I'm removing some fields and adding one, or maybe I can create one that both inherit from?